.github/workflows/test-demo.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+DISTRO="$1"
+NIXPKGS="https://github.com/NixOS/nixpkgs/archive/nixos-unstable.tar.gz"
```

Comment:
Wouldn't it be better to use the nixpkgs from our repo?

Reply:
This might make a sense to reduce number of nixpkgs in use. What's the best way to get nixpkgs version from ngipkgs without using flakes ?

Reply:
I suppose something like this (from the NGIpkgs root directory):

```shellSession
$ nix-instantiate --eval -A sources.nixpkgs.rev
"ebe4301cbd8f81c4f8d3244b3632338bbeb6d49c"
```

Reply:
Done now.